terraform/projects/app-ecs-services/nginx-auth-proxy.tf

```diff
@@ -0,0 +1,71 @@
+## IAM roles & policies
+
+### container, task, service definitions
+
+data "template_file" "nginx_auth_container_def" {
+  template = "${file("task-definitions/nginx-auth-proxy.json")}"
+
+  vars {
+    log_group = "${aws_cloudwatch_log_group.task_logs.name}"
+    region    = "${var.aws_region}"
+  }
+}
+
+resource "aws_ecs_task_definition" "nginx_auth_server" {
+  family                = "${var.stack_name}-nginx-auth"
+  container_definitions = "${data.template_file.nginx_auth_container_def.rendered}"
+  task_role_arn         = "${aws_iam_role.alertmanager_task_iam_role.arn}"
```

Comment:
why are we reusing the alertmanager role for the nginx task here?

Reply:
That is the wrong role, what is interesting is that everything is working regardless. This is because things are co-located and we are relying on the config grabber in the prometheus task. I am reluctant to add a config grabber task to the nginx because we don't need it but perhaps we should for consistency and in case the nginx gets scheduled on a box with out a config grabber.

Reply:
ew. i'm not massively keen on relying on the config grabber in the prometheus task